src/actions.sol

```diff
@@ -19,6 +19,7 @@ pragma solidity ^0.8.0;
 interface NFTLike {
     function approve(address usr, uint256 token) external;
     function transferFrom(address src, address dst, uint256 token) external;
+    function mintTo(address usr) external returns (uint256);
```

Comment:
```suggestion
    function mint(address usr) external returns (uint256);
```

The function we need to call here is just `mint()`

Reply:
@ilinzweilin seems like preferred another naming convention 
https://github.com/centrifuge/tinlake-asset-nft/blob/main/src/assetNFT.sol#L15

```solidity
   function mintTo(address usr) public auth returns(uint) {
        count++;
        _mint(usr, count); 
        return count;
    }```

Reply:
What is the point of the registry? @ilinzweilin

Reply:
For now, I'm going to deploy an actions contract with `mintTo()` that skips the registry and interacts directly with the NFT contract so that UI testing can proceed.

Reply:
@xmxanuel mintTo is the correct function to call... It is correct to use the AssetNFT contract here. The registry is just a minter contract on top...